java/arcs/core/util/Json.kt

```diff
@@ -152,6 +152,13 @@ sealed class JsonValue<T>() {
             JsonString(name).toString() + ":$value"
         }.joinToString(prefix = "{", postfix = "}", separator = ",")
 
+        constructor(vararg pairs: Pair<String, JsonValue<*>>) : this(
+            pairs.fold(mutableMapOf<String, JsonValue<*>>()) { map, pair ->
```

Comment:
this(pairs.associateBy { it.first }) would be more succinct

Reply:
pairs.associateBy({ it.first }, { it.second }) was what it needed to work. Updated now.